src/renderer/views/GridFeed.js

```diff
@@ -5,15 +5,28 @@ class GridFeed extends React.Component {
     constructor(props) {
         super(props);
         // pass awaitingMoreData as true to prevent lazy loading
-        this.state = { data: [], awaitingMoreData: props.awaitingMoreData }
+        let {getUser = undefined, awaitingMoreData = undefined} = props
+        this.state = { data: [], awaitingMoreData, getUser }
         this.handleScroll = this.handleScroll.bind(this);
     }
 
     componentDidMount() {
         window.addEventListener("scroll", this.handleScroll);
-        fetch(`https://3speak.online/apiv2/feeds/${this.props.type}`)
+        let url;
+        if (this.state.getUser) {
+            url = `https://3speak.online/apiv2/videoData?key=TEAM_IPFS&creator=${this.state.getUser}&start=0`
+        } else {
+            url = `https://3speak.online/apiv2/feeds/${this.props.type}`
+        }
+        fetch(url)
             .then(res => res.json())
             .then(json => {
+                if (json[0].timestamp) {
```

Comment:
Is there a particular reason why timestamp is being passed/renamed here?

If this is going to be passed to `VideoWidget.js` for displaying creation date, I believe that is already implemented [here](https://github.com/vaultec81/blasio-app/blob/6a4060d621f3604d60296e53a96c505fb21ade7a/src/renderer/components/video/VideoWidget.js#L57-L64)

Reply:
`timestamp` is what the API returns instead of `created` for the new request, the code you linked wouldn't work without this code.

Reply:
Oh I see my mistake I didn't see that you removed the piece of code which grabs the video info from caching db.